tasks/ci.rake

```diff
@@ -205,6 +205,46 @@ EOS
       end
     end
 
+    desc <<-EOS
+Setup acceptance tests using puppet-agent (AIO) packages.
+
+  $ SHA=<tag or full sha> HOSTS=<hosts> bundle exec rake ci:test:setup
+
+SHA should be the tag or full SHA for the puppet-agent package.
+
+HOSTS can be a beaker-hostgenerator string or existing file.
+EOS
+    task :setup => ['ci:check_env'] do |t, args|
+      sha = ENV['SHA']
+      case sha
+      when /^\d+\.\d+\.\d+$/
+        # tags are ok
+      when /^[0-9a-f]{40}$/
+        # full SHAs are ok
+      else
+        puts "Error: Expected SHA to be a tag or 40 digit SHA, not '#{sha}'"
+        exit 1
+      end
```

Comment:
This check should probably be moved to `ci:check_env` task (as long as it doesn't break existing usages for `rake ci:test:aio` or others)